src/pages/Beregning/Beregning.tsx

```diff
@@ -125,6 +126,21 @@ export function Beregning() {
     }
   }
 
+  React.useEffect(() => {
+    if (
+      isError ||
+      isInntektError ||
+      (alderspensjon && !alderspensjon?.vilkaarErOppfylt)
+    ) {
+      if (startAar && startAar < 67) {
```

Comment:
Jeg er usikker om denne logging blir helt riktig. Hvis isInntektError er true, og at brukeren har valgt en alder før 67 vil denne her logges, mens feilen skyldes ikke lav opptjening men feil i inntekt.
Kanskje bør det logges ulikt for:
- isInntektError
- !alderspensjon.vilkaarErOppfylt
- isError

Reply:
`isInntektError` vil aldri være false i Beregning. Fjernet logging for `isInntektLogging` og forenklet for Alert.